source/docs/software/wpilib-tools/glass/mech2d-widget.rst

```diff
@@ -0,0 +1,92 @@
+The Mechanism2d Widget
+======================
+
+Glass supports displaying stick-figure representations of your robot's mechanisms using the :guilabel:`Mechanism2d` widget. An instance of the ``Mechanism2d`` class should be created and populated, sent over NetworkTables, and updated periodically with the latest mechanism states in your robot code.
```

Comment:
What usefulness does showing stick figure representatives of mechanisms showcase? What mechanisms are supported? I'd like to expand this intro paragraph a bit.